GrindingCity.WebApi/Controllers/BuildingController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using GrindingCity.WebApi.DTOs;
+using GrindingCity.WebApi.Interfaces;
+using GrindingCity.WebApi.Models;
+using GrindingCity.WebApi.Repositories;
+using Microsoft.AspNetCore.Mvc;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace GrindingCity.WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BuildingController : ControllerBase
+    {
+        // GET all api
+        [HttpGet()]
+        public async Task<ActionResult<IEnumerable<Building>>> GetAllBuildings(
+            [FromServices] IBuildingRepository handler)
+        {
+            var result = await handler.GetAllBuildings();
+
+            return result == null ? NotFound(result) : Ok(result);

```

Comment:
Base on nullable refs its should always be false. Probably should be `!result.Any()`

Reply:
`NotFound(result)` should be without param